src/java/org/apache/cassandra/cql3/selection/Selection.java

```diff
@@ -126,6 +126,23 @@ public ResultSet.ResultMetadata getResultMetadata()
         return resultMetadata;
     }
 
+    public static Selection.Selectors noopSelector()
+    {
+        return new Selectors()
```

Comment:
Does `SimpleSelection` require a single table...and we may have more than one source table when building our result in `TransactionStatement#execute()`?

Reply:
Figured it would be simplest to do in feedback branch; was asking to avoid copy/paste by refactoring, did that in these 2 case; see aace70018a08bb9a8a3724804279665efbf473da

Reply:
when doing the refactor it was clear that the `isJson` stuff is a bit more annoying as where we use the `noop` wouldn't support json, so trying to pull that in didn't make much sense